src/Card.tsx

```diff
@@ -0,0 +1,27 @@
+import { cardAnatomy } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers } from "@chakra-ui/react";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(cardAnatomy.keys);
+
+const baseStyle = definePartsStyle({
+  container: {
+    backgroundColor: "#f4f4eb",
+    borderColor: "#d7da5e",
```

Comment:
NON-BLOCKING: Is there a reason you didn't use the semantic tokens for these? I'd use `brandGray` and `green` if possible.

Reply:
No, there is no reason, I will update it next time when I work on something else.